src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -248,7 +248,7 @@ private function displayGeneralText(SymfonyStyle $io, string $filter = null)
         }
     }
 
-    private function displayGeneralJson(SymfonyStyle $io, $filter)
+    private function displayGeneralJson(SymfonyStyle $io, string $filter = null)
```

Comment:
```suggestion
    private function displayGeneralJson(SymfonyStyle $io, ?string $filter)
```

Why do you introduce a default value?